src/sentry/release_health/duplex.py

```diff
@@ -446,7 +446,16 @@ def __init__(
     ):
         self.sessions = SessionsReleaseHealthBackend()
         self.metrics = MetricsReleaseHealthBackend()
-        self.metrics_start = metrics_start
+        self.metrics_start = min(
```

Comment:
```suggestion
        self.metrics_start = max(
```
We want to be able to set a more recent metrics_start, but never earlier than 90 days, right?

Reply:
yup thanks